src/calendar/view/CalendarView.ts

```diff
@@ -286,6 +292,9 @@ export class CalendarView extends BaseTopLevelView implements TopLevelView<Calen
 			ColumnType.Background,
 			size.second_col_min_width + size.third_col_min_width,
 			size.third_col_max_width,
+			undefined,
+			undefined,
+			true,
```

Comment:
is this `allowXOverflow`? Why do we need it? We should at least leave a comment

Reply:
A bit of background: Originally, it was just to display a line without a circle, but after a review of the code and functionality, it was decided to include the circle to keep it consistent with the other views (i.e. day/week view). This meant having to allow X overflowing, as the circle would clip outside of the list due to its negative offset.

I'll leave a comment later.